website/mails/mails.py

```diff
@@ -203,6 +203,31 @@ def get_english_article(word):
 
 # Merge account, add or remove email confirmation emails.
 CONFIRM_MERGE = Mail('confirm_merge', subject='Confirm account merge')
+COLLECTION_SUBMISSION_REJECTED = lambda collection, node: Mail(
+    'collection_submission_rejected',
+    subject=f'{node.title} was not accepted into {collection.provider.name}'
```

Comment:
You don't need the lambdas. These get parsed like mako templates and can use the context (see L535)

Reply:
It's just that this is consistent with the way the other subject lines are parsed.